matrimony_admin/views.py

```
@@ -1,30 +1,33 @@
+# Standard library imports
 import json
 import os
+from datetime import datetime, timedelta
```

Ruff

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)